tests/e2e/tslint.json

```diff
@@ -1,123 +1,124 @@
-          "isNull"
-        ],
-        [
-          "_",
-          "isDefined"
-        ]
+  "rules": {
+    "ban": [
+      true,
+      [
+        "_",
+        "extend"
       ],
-      "class-name": true,
-      "comment-format": [
-        true,
-        "check-space",
-        "check-lowercase"
+      [
+        "_",
+        "isNull"
       ],
-      "curly": true,
-      "eofline": true,
-      "forin": true,
-      "indent": [
-        true,
-        2
-      ],
-      "interface-name": true,
-      "jsdoc-format": true,
-      "label-position": true,
-      "label-undefined": true,
-      "max-line-length": [
-        false,
-        140
-      ],
-      "member-ordering": [
-        true,
-        "public-before-private",
-        "static-before-instance",
-        "variables-before-functions"
-      ],
-      "no-arg": true,
-      "no-bitwise": true,
-      "no-console": [
-        true,
-        "debug",
-        "info",
-        "time",
-        "timeEnd",
-        "trace"
-      ],
-      "no-construct": true,
-      "no-constructor-vars": true,
-      "no-debugger": true,
-      "no-duplicate-key": true,
-      "no-duplicate-variable": true,
-      "no-empty": true,
-      "no-eval": true,
-      "no-string-literal": true,
-      "no-switch-case-fall-through": true,
-      "no-trailing-comma": true,
-      "no-trailing-whitespace": true,
-      "no-unused-expression": true,
-      "no-unused-variable": true,
-      "no-unreachable": true,
-      "no-use-before-declare": true,
-      "no-var-requires": false,
-      "one-line": [
-        true,
-        "check-open-brace",
-        "check-catch",
-        "check-else",
-        "check-whitespace"
-      ],
-      "quotemark": [
-        true,
-        "single"
-      ],
-      "radix": true,
-      "semicolon": true,
-      "triple-equals": [
-        true,
-        "allow-null-check"
-      ],
-      "typedef": [
-        true,
+      [
+        "_",
+        "isDefined"
+      ]
+    ],
+    "class-name": true,
+    "comment-format": [
+      true,
+      "check-space",
+      "check-lowercase"
+    ],
+    "curly": true,
+    "eofline": true,
+    "forin": true,
+    "indent": [
+      true,
+      2
+    ],
+    "interface-name": true,
+    "jsdoc-format": true,
+    "label-position": true,
+    "max-line-length": [
+      false,
+      140
+    ],
+    "member-ordering": [
+      true,
+      "public-before-private",
+      "static-before-instance",
+      "variables-before-functions"
+    ],
+    "no-arg": true,
+    "no-bitwise": true,
+    "no-console": [
+      true,
+      "debug",
+      "info",
+      "time",
+      "timeEnd",
+      "trace"
+    ],
+    "no-construct": true,
+    "no-parameter-properties": false,
+    "no-debugger": true,
+    "no-duplicate-variable": true,
+    "no-empty": true,
+    "no-eval": true,
+    "no-string-literal": true,
+    "no-switch-case-fall-through": true,
+    "trailing-comma": [true,
+      {
+        "multiline": "always",
```

Comment:
@makambalaji : `"multiline": "always"` requirement looks redundant in case of objects, functions, typeliterals, etc.

WDYT about checking on arrays only?
```
"trailing-comma": [true, {
  "singleline": "never",
  "multiline": {
    "objects": "ignore", 
    "arrays": "always",
    "functions": "never",
    "typeLiterals": "ignore"
  }
}]
```

Reply:
I am ok with the change, but why do you want to ignore for objects, typeLiterals?

Reply:
"ignore" value for both objects, type literals. most of the current code doesn't have any impact.